Bottom: 5d8567651b8275bf1758541671b1945c62afa5d7
Top:    a6905793b5de800afb8cf1b42e0ace5122c129f5
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-06-08 17:29:15 -0700

Refresh of rs-1sge

---

diff --git a/src/rsocket.c b/src/rsocket.c
index 0f3bc91..fa16f85 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -756,6 +756,12 @@ static int rs_post_write(struct rsocket *rs, uint64_t wr_id,
 	return rdma_seterrno(ibv_post_send(rs->cm_id->qp, &wr, &bad));
 }
 
+static uint32_t rs_sbuf_left(struct rsocket *rs)
+{
+	return (uint32_t) (((uint64_t) (uintptr_t) &rs->sbuf[rs->sbuf_size]) -
+			   rs->ssge.addr);
+}
+
 /*
  * Update target SGE before sending data.  Otherwise the remote side may
  * update the entry before we do.
@@ -791,12 +797,6 @@ static int rs_write_data(struct rsocket *rs, struct ibv_sge *sge, int flags)
 	return ret;
 }
 
-static uint32_t rs_sbuf_left(struct rsocket *rs)
-{
-	return (uint32_t) (((uint64_t) (uintptr_t) &rs->sbuf[rs->sbuf_size]) -
-			   rs->ssge.addr);
-}
-
 static void rs_send_credits(struct rsocket *rs)
 {
 	struct ibv_sge ibsge;
